markdown/org/docs/patterns/carlita/fabric/en.md

```diff
@@ -1,2 +1,36 @@
-FIXME: Add fabric guidelines
+### Main Fabric
 
+You want a fabric that has a bit of body that holds its shape. Generally **Wools** of **coating weight** will be the best choice as they will provide warmth and give shape to the coat. Other fabrics of **coating weight** should also work. However if you are looking for something more lightweight you could try **Linens** and lighter **Wools**, but you may need to face these fabrics to give them more body before construction. **Fleece** is a cheaper alternate to **Wools** but may note keep as long, if you are putting the effort into properly tailoring Carlita then you are gonna want to consider buying a fabric that will last for a long period of time.
+
+<Note>
+
+Remember that this pattern is intended to be a winter coat so you need to consider warmth when choosing your fabric.
+
+</Note>
+
+### Linings and Pocket Bags
+
+Lightweight **Cottons** such as **Cotton Silesia**, **Cotton Lawn** or **Glazed Cottons** should do for the body lining but you may find that you require **Silks** or similar for the sleeve linings to reduce friction and make it easier to take on and off over the arms. **Silks** can also be used for the body as well but the **Cotton** will be more durable in the long run. Pocket bags should be out of the **Cotton** or something that is strong enough to hold the weight of the intended items you wish to carry in them. **Printed Cottons** are a fun thing to try but if your main fabric is quite bulky you may find they add to much weight. 
+
+<Tip>
+
+As per usual with Linings it is really down to your prefered lining materials but if you are new to lining a coat we suggest to use the **Cotton** options for the body and pockets and use the **Silk** option for the sleeves.
+
+</Tip>
+
+### Facings
+
+**Hair canvases**, you may be tempted to use modern interfacing but hair canvases will provide the best results. You will want a combination of different weight hair canvases such as lightweight to medium weight **Tailors Canvas** and heavyweight **Horsehair Canvas**.
```

Comment:
Maybe "Hair canvases" was meant to be a heading? I'm not sure how to read it as an intro.

Reply:
No but I can see the wording is off give me a sec.